_posts/2017-12-19-fuzzy-search-in-elixir.md

```diff
@@ -0,0 +1,284 @@
+---
+layout: post
+title:  "Implementing Fuzzy Search in Elixir Using Ecto and Postgres"
+date:   2017-12-19 12:00:00 -0400
+categories: api
+author: Demi Filippou
+---
+
+## Background
+Here at Forward Financing, our Elixir single sign on app has always stored only a small number of users. We developed only internal tools and therefore only kept track of internal users. However, now that we are expanding and developing external facing tools, the number of users is skyrocketing. This caused a problem for admins who were creating, editing, and checking for users. To check if a user exists, an admin would have to manually click through the growing number of pages of users. Yuck. That is why this sprint, our project managers decided it was time to implement search functionality for users. We decided to go down the route of fuzzy search, because why vanilla search when you can fuzzy search?
```

Comment:
single sign-on

Reply:
~That is why~ ~t~`T`his sprint

Reply:
I think you mean `Some may say this sprint`?